performance/populate_database/populate_database.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/csv"
+	"flag"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/cloudspannerecosystem/harbourbridge/performance"
+	"github.com/go-sql-driver/mysql"
+)
+
+const (
```

Comment:
Please add some docs here outlining
1. Purpose of the module
2. How and when to execute it.

Reply:
Sure, makes sense added a readme file